src/test/java/task_at_7/CalculatorTest.java

```diff
@@ -0,0 +1,39 @@
+package task_at_7;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class CalculatorTest {
+
+
+    @Test
+    public void mathOperationAdd() {
+        Calculator calculator = new Calculator();
```

Comment:
why don't you create one instance of Calculator in terms of CalculatorTest class?

Reply:
Got it. It can be created once for all tests.